#Camera Movement
float32 dX
float32 dY
float32 dZ 

#Camera Angle
float32 angleX
float32 angleY
float32 angleZ 


